### ISSUE TITLE
[Suggestion]: Where can I find resources on how to use Statement for my mod?

### ISSUE THREAD

**Issue by Felicis** [Virtuoel/Statement#25]
### What is your suggestion?

TL;DR: I'm trying to add WATERLOGGING and a new custom BlockStateProperty to as many blocks as possible using Statement, but just adding the block properties to a block using the StateRefresher.INSTANCE does not seem to work. How can I get this to work? Where can I get help? Is there documentation?

I've just started modding and recently adopted the (forge) mod [water physics](https://www.curseforge.com/minecraft/mc-mods/water-physics-overhaul) which completely overwrites the vanilla water mechanics to create more realistic fluid flowing, equalization and displacement mechanics, where water is no longer infinite and fluids move about in 125 mb packets (a new BlockStateProperty FLUID_LEVEL stores the # of fluid packets in one blockstate).

Obviously the nicest thing would be to have water flow around and through as many blocks as possible, but the current system has many glitches with the vanilla waterlogging mechanics and is completely incompatible with modded blocks. 
In an issue on the original repo someone advised to use Statement to add fluid waterlogging to vanilla and modded blocks which are not waterlogged yet and I frankly I believe the new FLUID_LEVEL property should be added that way too.

However I am quite confused as how to use the mod to achieve said goals. I have had a look at mods using Statement (FlowerTweaks, Canals-v2 and Towelette), but while the simple changes from the first two mods (only adding the block properties to the block using the Statement Refresher INSTANCE) do not seem to work, I am completely overwhelmed with what is done in towelette (although it looks like a really good approach. However Towelette is fabric, and this mod is forge, so I cannot run it together and outsource the waterlogging to towelette.

Since both Statement and Towelette are your mods and I did not find a good resource on how to use those mods, I though to write here and ask you for help or guidance. 

Felicis
